src/hres_ic.py

```diff
@@ -0,0 +1,63 @@
+#!/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.01/bin/python
+
+"""
+Replace the land/surface fields in the astart file with higher-resolution
+era5-land or BARRA2-R data (if requested).
+"""
+
+from pathlib import Path
+import argparse
+from datetime import datetime,timedelta
+import pandas
+import shutil
+
+import replace_landsurface_with_ERA5land_IC 
+import replace_landsurface_with_BARRA2R_IC
+
+boolopt = {
+    "True": True,
+    "False": False,
+}
```

Comment:
Similarly here - it doesn't seem to be referenced anywhere else in the PR...

Reply:
Same as above.

Reply:
No worries!